src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -3,33 +3,55 @@ title: Dynamic Sampling
 
-With dynamic sampling, you control which transactions Sentry retains by setting _server-side sampling_ rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In addition, dynamic sampling allows you to manage your sampling configuration centrally, server-side, and lets you make changes without the need to reconfigure the SDK and redeploy your application.
+| Prioritize | Ignore |
+|------------|--------|
+| Latest release | Health checks |
+| Dev environments | |
 
-![The Dynamic Sampling page with a list of rules](dynamic-sampling.png)
+<!-- prettier-ignore-end -->
 
-> _[A trace](/product/sentry-basics/tracing/distributed-tracing/) represents the record of an entire operation you want to measure or track, like a page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and the trace can be distributed across multiple services or projects._
+We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meanginful insights during your testing phase, we prioritize transaction data from these environments.
 
-Dynamic sampling allows you to retain a smaller number of more relevant transactions that you can explore in full detail, without giving up the accuracy of performance metrics that are based on larger set of all the transactions you send to Sentry.
+_Health check_ type transactions, while important for checking the stability of your application, don't have any value for you beyond the task associated with them. They also tend to be high in volume, potentially using up a lot of you quota if retained. To avoid using up your volume on this type of transactions, we ignore these when sampling your data.
 
-Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more transactions from high converting pages, critical API endpoints, or you may need to focus on latency issues from your latest release.
+While we do our best to provide you with the most useful transaction data, our **Dynamic Sampling** feature allows you to turn the priorities that we apply to your data retention on or off:
 
-Dynamic sampling also makes your SDK (client-side) and your server-side sampling configuration largely independent of each other. This helps if you're building native applications with many active releases at a given time.
+![The Dynamic Sampling page with priority toggles enabled.](dynamic-sampling.png)
 
+<!-- placeholder and needs to be updated before PR is merged -->
+
+Because every project is different, you can set these toggles differently for different projects. For example, you may have a project where the retaining transactions from the latest release is not a priority for you.
```

Comment:
```suggestion
Because every project is different, you can set these toggles differently for different projects. For example, you may have a project where retaining transactions from the latest release is not a priority for you.
```